src/_data/header.yml

```diff
@@ -21,3 +21,8 @@
   icon: blog
   mobileIcon: blog
   blogurl: true
+# - name: Enterprise
```

Comment:
What are the blockers in your mind to uncommenting this?

Reply:
I need an icon for Enterprise, and didn't want that to hold up this PR. I'll merge this one now and create a separate PR now to uncomment the menu item.